File: PspDataPresenter.Core/Models/MeetingModel.cs
using System;
using System.Collections.Generic;

namespace PspDataPresenter.Core
{
    public class MeetingModel
    {
        public int Id { get; set; }
        public int MeetingNo { get; set; }
        public bool CanRemotelySignUpForDebate { get; set; }
        public BoardModel Board { get; set; }
        public RoomModel Room { get; set; }
        public CurrentAgendaItem CurrentAgendaItem { get; set; }
        public MajorityModel Majority { get; set; }
        public List<AgendaModel> Agenda { get; set; }
        public List<RemarkModel> Remarks { get; set; }
        public List<FollowingItemModel> FollowingItems { get; set; }
        public DateTime Timestamp { get; set; }
    }
}

File: PspDataPresenter.App/data-presenter-app/src/app/_core/models/meeting.model.ts
import { AgendaModel } from "./agenda.model";
import { BoardModel } from "./board.model";
import { CurrentAgendaItem } from "./current-agenda-item.model";
import { FollowingItemModel } from "./following-item.model";
import { MajorityModel } from "./majority.model";
import { RemarkModel } from "./remark.model";
import { RoomModel } from "./room.model";

export interface MeetingModel
{
    id: number;
    meetingNo: number;
    canRemotelySignUpForDebate: boolean;
    board: BoardModel;
    room: RoomModel;
    currentAgendaItem: CurrentAgendaItem;
    majority: MajorityModel;
    agenda: AgendaModel[];
    remarks: RemarkModel[];
    followingItems: FollowingItemModel[];
    timestamp: Date | string;
}